Makefile

```diff
@@ -0,0 +1,25 @@
+include .bootstrap.mk
+
+DOCKERFILES := $(shell ls -1 *.Dockerfile)
```

Comment:
Do not use `ls` for file listings:

1. Use globbing in Bash
2. Use the wildcard function in Make: https://www.gnu.org/software/make/manual/html_node/Wildcard-Function

Reply:
done.